Description: Debian specific tweaks for NetworkManager systemd service file
 - Install an alias named network-manager.service as the legacy SysV init script
   in Debian is called network-manager.
 - Do not install NetworkManager into the network.target, as this will lead to a
   dependency loop (NetworkManager is installed in /usr).
Author: Michael Biebl <biebl@debian.org>
Index: network-manager/data/NetworkManager.service.in
===================================================================
--- network-manager.orig/data/NetworkManager.service.in	2012-03-02 21:43:15.458064485 +0100
+++ network-manager/data/NetworkManager.service.in	2012-03-02 21:43:20.242064316 +0100
@@ -1,8 +1,6 @@
 [Unit]
 Description=Network Manager
 After=syslog.target
-Wants=network.target
-Before=network.target
 
 [Service]
 Type=dbus
@@ -16,3 +14,4 @@
 [Install]
 WantedBy=multi-user.target
 Alias=dbus-org.freedesktop.NetworkManager.service
+Alias=network-manager.service
